scrc/utils/court_names.py

```diff
@@ -0,0 +1,53 @@
+court_names_bu = ['AG_AK', 'AG_LWRK', 'AG_OG', 'AG_OGA', 'AG_OSB', 'AG_PRG', 'AG_RGAR', 'AG_SKBG', 'AG_SRG', 'AG_SVWG', 'AG_VB', 'AG_XX', 'AI_BZG', 'AI_KG', 'AI_XX', 'AR_KG', 'AR_OG', 'AR_RR', 'AR_SRK', 'AR_VB', 'AR_XX', 'BE_AK', 'BE_NAB', 'BE_OG', 'BE_SRK', 'BE_VB', 'BE_VG', 'BE_XX', 'BL_EG', 'BL_KG', 'BL_SG', 'BL_XX', 'BL_ZMG', 'BS_APG', 'BS_SVG', 'BS_XX', 'CH_BGE', 'CH_BGer', 'CH_BSTG', 'CH_BVGE', 'CH_EDÖB', 'CH_PATG', 'CH_VB', 'CH_WBK', 'CH_XX', 'FR_TAMA', 'FR_TC', 'FR_XX', 'GE_CAPJ', 'GE_CJ', 'GE_TAPI', 'GE_TP', 'GE_XX', 'GL_KG', 'GL_OG', 'GL_VG', 'GL_XX', 'GR_KG', 'GR_VG', 'GR_XX', 'JU_TC', 'JU_TPI', 'JU_XX', 'LU_AUK', 'LU_BKD', 'LU_BZG', 'LU_GSD', 'LU_JSD', 'LU_KG', 'LU_RR', 'LU_RSH', 'LU_XX', 'NE_ARA', 'NE_ASL', 'NE_ASS', 'NE_ATS', 'NE_CA', 'NE_CN', 'NE_TC', 'NE_TR', 'NE_XX', 'NW_OG', 'NW_XX', 'OW_OG', 'OW_VB', 'OW_VG', 'OW_XX', 'SG_ABSK', 'SG_HG', 'SG_KG', 'SG_KGN', 'SG_VB', 'SG_VG', 'SG_VGN', 'SG_VSG', 'SG_VWEK', 'SG_XX', 'SH_OG', 'SH_XX', 'SO_OG', 'SO_SK', 'SO_STG', 'SO_VG', 'SO_VSG', 'SO_XX', 'SZ_KG', 'SZ_XX', 'TG_OG', 'TG_XX', 'TI_CARP', 'TI_CATI', 'TI_CRP', 'TI_GIAR', 'TI_GPC', 'TI_PP', 'TI_TCA', 'TI_TCAS', 'TI_TE', 'TI_TPC', 'TI_TRAC', 'TI_TRAP', 'TI_TRPI', 'TI_XX', 'UR_REB', 'UR_XX', 'VD_SR', 'VD_TC', 'VD_TN', 'VD_TPHA', 'VD_XX', 'VS_AG', 'VS_BZG', 'VS_SRK', 'VS_TC', 'VS_XX', 'ZG_VG', 'ZG_XX', 'ZH_BK', 'ZH_BRK', 'ZH_HG', 'ZH_KSG', 'ZH_OG', 'ZH_SOBE', 'ZH_SRK', 'ZH_SVG', 'ZH_VG', 'ZH_XX', 'ZG_UPL', 'BE_UPL', 'SG_OG', 'FR_UPL', 'VD_UPL', 'ZH_UPL', 'AG_JG', 'AG_VG', 'AG_HG', 'AG_VSG', 'LU_UPL', 'CH_UPL', 'AG_UPL', 'AG_AUK', 'AG_RR', 'AG_JL', 'AI_UPL', 'AR_UPL', 'BL_UPL', 'BS_UPL', 'GE_UPL', 'GL_UPL', 'GR_UPL', 'JU_UPL', 'NE_UPL', 'NW_UPL', 'OW_UPL', 'SG_UPL', 'SH_UPL', 'SO_UPL', 'SZ_UPL', 'TI_UPL', 'TG_UPL', 'UR_UPL', 'VS_UPL', 'UR_OG']
+
+
+def get_all_courts():
+    return court_names_bu
+    try:
+        court_names = next(self.select(engine, "court", "court_string", None))["court_string"].tolist()
```

Comment:
this is never executed.

Reply:
Ohh, I realise this code isn't done yet.
For using the self.select and the engine to get the court names from the DB I should probably move this to the dataset_creator, right? 
What do you think?

Reply:
yes, I think that would make sense. I like that you are splitting up code into different files. In this case, however, I think it might make more sense to refactor out some functions from the dataset creator. Examples that come to mind are the plotting functions or the saving functions or retrieving functions.

Reply:
Yeah, I see. Especially the plotting functions should be fairly easy to refactor out since they're not an instance of the DatasetCreator class.

Reply:
exactly